api/v1alpha1/reaper_types.go

```diff
@@ -26,7 +26,8 @@ import (
 type StorageType string
 
 const (
-	DefaultReaperImage = "thelastpickle/cassandra-reaper:2.1.3"
+	DefaultReaperImage     = "thelastpickle/cassandra-reaper:2.1.3"
+	DefaultImagePullPolicy = "ifNotPresent"
```

Comment:
I think this needs to be capitalized , `i`fNotPreset --> `I`fNotPresent

Reply:
It wasn't actually being used at all (validate used corev1 directly), but I'll modify the behavior.